src/index.js

```diff
@@ -30,10 +30,19 @@ const getPrefix = ({
 }
 
 const getId = (path, prefix) => {
-  if (!(path.isIdentifier() && path.node.name)) {
-    throw new Error(`require Object key`)
+  let name;
+  
+  if (path.isStringLiteral()) {
+    name = path.node.value
+  } else if (path.isLiteral()) {
```

Comment:
`path.isLiteral()` → `path.isIdentifier()`

```js
const messages = defineMessages({
  title: '...', // ← Identifier
  'title2': '...' // ← StringLiteral
})
```


Please check this.
https://astexplorer.net/#/gist/28061859b30ad2cd7aef75ee70aed767/latest

Reply:
Sorry about that. Bad copy paste.